homeworks/homework-1/app/src/components/LoginForm.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+	Button,
+	Form,
+	FormGroup,
+	FormControl,
+	ControlLabel,
+	Col
+} from "react-bootstrap";
+
+class LoginForm extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			hintEmail: "*enter your email",
```

Comment:
if text will not change this is no reason to store that in `state` or if you wanna to reuse this text across component specify them as consts in `render` method